sql/api/pom.xml

```diff
@@ -35,6 +35,16 @@
     </properties>
 
     <dependencies>
+        <dependency>
+            <groupId>org.apache.spark</groupId>
+            <artifactId>spark-unsafe_${scala.binary.version}</artifactId>
```

Comment:
Why do we need unsafe here?

Reply:
Can you put `ByteArrayMethods.MAX_ROUNDED_ARRAY_LENGTH` in common/util?

Reply:
good point. Yeah let me try so that we do not add too much deps to `sql/api`.

Reply:
done